### PR TITLE
add german chefkoch.de

### PR DESCRIPTION
**Pull request by rndmh3ro**
test url: https://www.chefkoch.de/rezepte/312821112629509/Hackfleischrouladen.html

result:
grafik

### CODE REVIEW

File: parsers/chefkoch.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class Chefkoch(Recipe):
+
+    def get_json_recipe(self, r):
+        recipe = {}
+
+        recipe['name'] = r['name']
+        recipe['description'] = r['description']
+        recipe['ingredients'] = r['recipeIngredient']
+        recipe['instructions'] = [r['recipeInstructions']]
```
Comment (poundifdef) [inline parsers/chefkoch.py:12]: Thank you for this! Might I suggest one change: It looks like, in the `recipeInstructions`, each individual instruction is separated by `\r\n`. For example, this recipe: https://www.chefkoch.de/rezepte/1474841252484744/Spare-Ribs-zum-Grillen-oder-fuer-den-Backofen.html

If your code where to split `r['recipeInstructions']` based on newlines, then the instructions could be numbered. As written, the above example shows as a single paragraph.

If you would be willing to make that update, then happy to merge this!

Comment (rndmh3ro) [reply parsers/chefkoch.py:12]: Very good idea, thanks! I implemented it.

Comment (rndmh3ro) [reply parsers/chefkoch.py:12]: Well, not quite. The linebreaks differ in the recipes.

Comment (hotfix) [reply parsers/chefkoch.py:12]: Chefkoch is actually not that easy as there are no explicit steps defined
here are a few recipes where the steps are different

https://www.chefkoch.de/rezepte/1218391227356456/Der-beste-Kaesekuchen-der-Welt.html
between steps is double \n

https://www.chefkoch.de/rezepte/1255131230975627/Traenenkuchen-der-beste-Kaesekuchen-der-Welt.html
between steps is double \r\n

https://www.chefkoch.de/rezepte/312821112629509/Hackfleischrouladen.html
only one \r\n


https://www.chefkoch.de/rezepte/2060231333350946/Auberginen-Tomaten-Schichtsalat.html
one more exotic case

Comment (rndmh3ro) [reply parsers/chefkoch.py:12]: Regex to the rescue: 
grafik
grafik
grafik
grafik

This should catch all newlines (if there are any).

**Comment by poundifdef** [comment]
Nice work! Thanks so much for iterating on this.

**Pull request merged**